runtime/android/core_internal/src/org/xwalk/core/internal/XWalkSettings.java

```diff
@@ -718,6 +721,44 @@ private boolean getSaveFormDataLocked() {
         return mAutoCompleteEnabled;
     }
 
+    void setDIPScale(double dipScale) {
+        synchronized (mXWalkSettingsLock) {
+            mDIPScale = dipScale;
+            // TODO(joth): This should also be synced over to native side, but right now
+            // the setDIPScale call is always followed by a setWebContents() which covers this.
+        }
+    }
+
+    /**
+     * See {@link android.webkit.WebView#setInitialScale}.
```

Comment:
The comment was documented at https://github.com/wuhengzhi/crosswalk/blob/b7901a51c540a13db49845081dd4797a3f72184b/runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java#L914, do you mean this?